### DISCUSSION TITLE
Saludos

### DISCUSSION THREAD

**Discussion by seguis** [upptime/upptime#602] [Q&A] [answered]
Amigo tienes WhatsApp o telegram, tengo varias dudas sobre linux

**Comment by AnandChowdhary** [comment, marked as answer]
Unfortunately we don't have Telegram or WhatsApp, but you can post any questions here in Github Discussion!

Translated by Google:

> Desafortunadamente, no tenemos Telegram ni WhatsApp, ¡pero puedes publicar cualquier pregunta aquí en Github Discussion!
